### PR TITLE
Update Change Log for SE-0075

### PR DESCRIPTION
**Pull request by CodaFi**
Some due diligence on my part.  This made the cut for 4.1.

### CODE REVIEW

**Comment by CodaFi** [comment]
@swift-ci please smoke test

File: CHANGELOG.md [hunk 1]
```diff
@@ -47,6 +47,25 @@ Swift 5.0
 Swift 4.1
 ---------
 
+* [SE-0075][]
+
+  Compile-time testing for the existence and importability of modules is now
+  implemented as a build configuration test.  The `#canImport` test allows
```
Comment (jrose-apple) [inline CHANGELOG.md]: Nitpick: `canImport` or `#if canImport`, not `#canImport`.

**Comment by CodaFi** [comment]
@swift-ci please smoke test and merge

**Comment by CodaFi** [comment]
@swift-ci please smoke test and merge

**Pull request merged**